secp256k1-haskell-recovery/package.yaml

```diff
@@ -0,0 +1,44 @@
+name: secp256k1-haskell-recovery
+version: 0.1.0
+synopsis: Bindings for recoverable signatures feature of secp256k1
+description: Sign and verify recoverable signatures using the secp256k1 library.
+category: Crypto
+author: Jean-Pierre Rupp
```

Comment:
This is your package. You can put your name here instead of mine.

Reply:
Done